packages/oas/medusa-oas-cli/src/command-oas.ts

```diff
@@ -17,6 +17,14 @@ import {
 const medusaPackagePath = path.dirname(
   require.resolve("@medusajs/medusa/package.json")
 )
+// Types package directory
+const medusaTypesPath = path.dirname(
+  require.resolve("@medusajs/types/package.json")
+)
+// Utils package directory
+const medusaUtilsPath = path.dirname(
+  require.resolve("@medusajs/utils/package.json")
+)
```

Comment:
I've asked him yesterday to have a look.

Reply:
LGTM. As I mentioned to Carlos:
> I also checked that @medusajs/types and @medusajs/utils where part of the dependency graph. Since they are a dependency of the core package we should be fine.
Plus, the OAS github check didn’t fail and we don’t see any changes to the generated/client-types package.

Reply:
now, after a couple develop merges, the OAS unit tests are failing. any clue why?

Reply:
this is odd. They weren't working haha. I wouldn't expect unit tests to have intermittent errors

Reply:
@carlos-r-l-rodrigues they normally don't, it is generally something different that trigger those tests to fail